v3/src/components/common/edit-formula-modal.tsx

```diff
@@ -47,6 +52,7 @@ export const EditFormulaModal = observer(function EditFormulaModal({
     setShowFunctionMenu(false)
     setFormula(value || "")
     onClose?.()
+    setDimension({ width: minWidth, height: minHeight })
```

Comment:
Are we sure we want to reset the size on close? I can see an argument that retaining its size across invocation would be desirable.